decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,93 @@
+# 1. Record architecture decisions
+
+Date: 2018-10-12
+
+## Status
+
+Accepted
+
+## Context
+
+During our work on Manage Offenders in Custody, we will have to make
+architectural decisions around both design and scope of the work.
+
+When making decisions, we should record them somewhere for future reference,
+to help us remember why we made them, and to help teams working in related areas
+understand why we made them.
+
+We should make our decisions public so that other teams can find them more
+easily, and because [making things open makes them better](https://www.gov.uk/guidance/government-design-principles#make-things-open-it-makes-things-better).
+
+Work done as a result of decisions we need to record is likely to be split
+across multiple repositories.
+
+## Decision
+
+We will use Architecture Decision Records, as described by Michael Nygard in
+this article: http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions
+
+An architecture decision record is a short text file describing a single
+decision.
+
+We will keep ADRs in this public repository under decisions/[number]-[title].md
+
+We should use a lightweight text formatting language like Markdown.
+
+ADRs will be numbered sequentially and monotonically. Numbers will not be
+reused.
+
+If a decision is reversed, we will keep the old one around, but mark it as
+superseded. (It's still relevant to know that it was the decision, but is no
+longer the decision.)
+
+We will use a format with just a few parts, so each document is easy to digest.
+The format has just a few parts.
```

Comment:
@jennyd - This line seems to be a duplication, so could be removed?
```suggestion
The format has just a few parts.
```

Reply:
Good spot! I've changed the . to a : at the end of the previous line too.